src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -2,25 +2,27 @@
 import { _, $, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals//BaseFooter';
 import Logger from '../../../util/Logger';
 
 const request = (url, method = 'GET', body = null, timeout = 1000) => {
-  return $.ajax({
+  let ajaxOpt = {
     url,
     method,
-    timeout,
     contentType: 'application/json',
     data: body,
-  });
+  };
+  if (typeof timeout === 'number') {
```

Comment:
why do we need this check ?

Reply:
jquery would not use any undefined `timeout` value hence this check it's redundancy if your intent to avoid overwrite `timeout` to `undefined`